src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeErrorsTest.java

```diff
@@ -0,0 +1,266 @@
+        records.add(json);
+
+        JsonObject root = new JsonObject();
+        root.put("records", records);
+
+        WebClient client = WebClient.create(vertx);
+
+        client.post(BRIDGE_PORT, BRIDGE_HOST, "/topics/" + kafkaTopic)
+                .putHeader("Content-length", String.valueOf(root.toBuffer().length()))
+                .as(BodyCodec.jsonObject())
+                .sendJsonObject(root, ar -> {
+                    context.assertTrue(ar.succeeded());
+
+                    HttpResponse<JsonObject> response = ar.result();
+                    JsonObject bridgeResponse = response.body();
+
+                    JsonArray offsets = bridgeResponse.getJsonArray("offsets");
+                    context.assertEquals(1, offsets.size());
+                    int code = offsets.getJsonObject(0).getInteger("error_code");
+                    String statusMessage = offsets.getJsonObject(0).getString("error");
+
+                    context.assertEquals(ErrorCodeEnum.PARTITION_NOT_FOUND.getValue(), code);
+                    context.assertEquals("Invalid partition given with record: 1000 is not in the range [0...3).", statusMessage);
+                    async.complete();
+                });
+    }
+
+    //@Test
+    public void sendToNonExistingTopicTest(TestContext context) {
+        String kafkaTopic = "sendToNonExistingTopicTest";
+
+        Async async = context.async();
+
+        String value = "Hi, This is kafka bridge";
+        int partition = 1;
+
+        JsonArray records = new JsonArray();
+        JsonObject json = new JsonObject();
+        json.put("value", value);
+        json.put("partition", partition);
+        records.add(json);
+
+        JsonObject root = new JsonObject();
+        root.put("records", records);
+
+        WebClient client = WebClient.create(vertx);
+
+        client.post(BRIDGE_PORT, BRIDGE_HOST, "/topics/" + kafkaTopic)
+                .putHeader("Content-length", String.valueOf(root.toBuffer().length()))
+                .as(BodyCodec.jsonObject())
+                .sendJsonObject(root, ar -> {
+                    context.assertTrue(ar.succeeded());
+
+                    HttpResponse<JsonObject> response = ar.result();
+                    JsonObject bridgeResponse = response.body();
+
+                    JsonArray offsets = bridgeResponse.getJsonArray("offsets");
+                    context.assertEquals(1, offsets.size());
+                    int code = offsets.getJsonObject(0).getInteger("error_code");
+                    String statusMessage = offsets.getJsonObject(0).getString("error");
+
+                    context.assertEquals(ErrorCodeEnum.TOPIC_NOT_FOUND.getValue(), code);
+                    context.assertEquals("Topic " + kafkaTopic + " not found", statusMessage);
+                    async.complete();
+                });
+    }
+
+    //@Test
+    public void sendToNonExistingTopicNonExistingPartitionTest(TestContext context) {
```

Comment:
instead of commenting, let's use @Ignore so that we have trace during the tests that something is ignored and it the future we can re-activate that.